.cspell.json

```diff
@@ -1,21 +1,22 @@
 {
```

Comment:
you dont need to add the .cspell.json file. so please remove it

Reply:
Okay will do. Thank you.